terraform/environments/corporate-staff-rostering/locals_preproduction.tf

```diff
@@ -86,6 +89,8 @@ locals {
           create_internal_record = true
           create_external_record = true
         }
+        disable_api_termination  = true
```

Comment:
This is already declared in the `instance` argument above. These should be in that block.

Reply:
resolved